Moth Food Returns: A Taste of the Fleet

EOBGames opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
![image](https://user-images.githubusercontent.com/58124831/144130513-2dc8f94f-8025-4137-a53f-d48aefa1536b.png)
Well, I figure it's time to make good on that promise.
In this thrilling sequel to #59412, Moths get the culinary overhaul they've needed for a while.
Adds somewhere in the region of 50 new foods, all tied together by a connection to Moths and their culture, as well as around 10 new drinks.
![image](https://user-images.githubusercontent.com/58124831/135012623-2f882da4-3ac9-4ddf-890f-cf22fe1de3c4.png)
As with #59412, this PR has three aims:
1. Introduce moth culture (as it exists in Common Core lore) to the game in a tangible way.
2. Give further variety to the chef and bartender, including some form of new cooking challenge for them.
3. To give some flavour (metaphorically and literally).

![image](https://user-images.githubusercontent.com/58124831/135012799-f3eb7183-9914-4057-81e9-03353be0edea.png)
This time, I actually will include a complete list of the new foods and their recipes: https://hackmd.io/tFr8ef46TeeF5HkDzxxIhw?view

As a nomadic people, moths have a wide variety of foods from various cultures. In particular, they've adopted a lot from humanity. Expect a mix of stranger and more distinctly alien foods, along with some human classics that have underwent a mothic transformation. Additionally, any language fans may enjoy the comments on each Mothic named food item, giving a glimpse into Mothic as a language.
![image](https://user-images.githubusercontent.com/58124831/135012988-fbb2df53-9a8d-462f-826b-97216c39c817.png)
Töchtaüse Berries
![image](https://user-images.githubusercontent.com/58124831/135013058-69a63013-d96d-4225-ad75-da68e98d3d68.png)
A popular food on the Mothic Fleet, töchtaüse berries are well known for being deeply unpleasant while raw: they're spicy, bitter, and filled with a mild poison that causes itching. Once heated to a sufficient temperature and mixed with sugar, however, they're a complex flavour that adds nicely to both sweet and savoury dishes.

Psst. Hey. Secret extra feature for secret fans:
![image](https://user-images.githubusercontent.com/58124831/135025322-4ae42247-cdc2-4b67-a0f4-be529e96d01a.png)
This charming poster, originally by Capsandi, from #51867. Moffuchi's may be gone, but it's not forgotten.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
The lizard food PR was a good first step in making Common Core lore better integrated with the game, but it was only that: a start. This is a continuation that should keep that trend going and making the universe deeper.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. -->

:cl:
add: The release of the new cookbook, "Mothic Cuisine: of Fleet and Flotilla" has brought Mothic food to the masses! Try out some new treats, like cheesecake balls, cheese soup, and five-cheese pizza at the kitchen today! There might even be some foods that don't have cheese, too!
add: To coincide with the new popularity of Mothic cuisine, Nanotrasen now stocks seeds for Töchtaüse Berries, a common ingredient in moth cooking, in the Megaseed vendors. Additionally, herbs and bell peppers have also made their way to the station's stock. Botanists, you know what to do.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

EOBGames committed:
moth food returns again

EOBGames committed:
moth food is back

EOBGames commented:
And of course it's conflicting immediately, for fucks sake I can't escape them with this content

Googles-Hands commented:
Bro didn't you hear, we aren't supposed to add more bloat clothes, I know you like moths but eh

Farquaar commented:
No cloth food?

EOBGames committed:
Merge remote-tracking branch 'upstream/master' into I'm-all-mothed-out
resolve conflicts on #63174

optimumtact commented:
![handsclaspedglasses](https://user-images.githubusercontent.com/2544323/144134314-36316ddb-1e66-4da4-a6cf-3627eef1d77a.png)

CRITAWAKETS commented on code/game/objects/items/food/misc.dm:
```diff
@@ -650,6 +650,25 @@
 
+/obj/item/food/canned/tomatoes
+	name = "canned San Marzano tomatoes"
+	desc = "A can of premium San Marzano tomatoes, from the hills of Southern Italy."
```
Should be something other than a real location, like Space Italy.

EOBGames replied:
Not everything needs to be from space, especially not something with a DOP.

CRITAWAKETS commented on code/game/objects/items/food/moth.dm:
```diff
@@ -0,0 +1,884 @@
+	w_class = WEIGHT_CLASS_SMALL
+
+//Base ingredients and miscellany, generally not served on their own
+/obj/item/food/grilled_cheese
```
wait dont we already have grilled cheese?

EOBGames replied:
This is grilled cheese as in a grilled piece of cheese, a la halloumi, not a grilled cheese sandwich. The description quite clearly states that, I think.

CRITAWAKETS replied:
oh im dumb, carry on

EricZilla replied:
You bout to add moth halloumi???

CRITAWAKETS commented:
Items like mozzarella cheese, tomato/pesto sauce and the likes should probably be somewhere more towards what they are than the moth food file.

EOBGames committed:
a variety of fixes and changes

made the new additions craftable, added cargo crates, and made some misc foods misc foods rather than moth foods

EOBGames committed:
fix bell peppers?

carshalash commented:
Told you that you'd get it man. I knew you could do it eob chan.

Zytolg commented:
POUTINE RETURNS

EOBGames committed:
some minor changes

EOBGames committed:
donk dinner

EOBGames commented:
I'm going to draft this since there's some stuff needing changed around- it shouldn't take long but I don't want to risk this getting merged while it's not done

EOBGames committed:
ready donk

EOBGames committed:
done?

EOBGames commented:
And with that I believe this should be ready for review.

Kylerace reviewed (approved):
code is good, your life is in the spritetainers hands

Imaginos16 reviewed (approved):
Sprites are good, your life is now on the maintainers ha- Oh wait, they already approved!

tuckerorchistra commented:
Why would moths cook regular food if they eat cloth.

Mothblocks commented:
why do you cook regular food if you eat bread
